src/coreclr/jit/lsrabuild.cpp

```diff
@@ -3256,24 +3256,66 @@ void LinearScan::setDelayFree(RefPosition* use)
     pendingDelayFree  = true;
 }
 
+//------------------------------------------------------------------------
+// DelayFreeUses: Mark useRefPosition as delay-free, if applicable for the
+//                rmw node.
+//
+// Arguments:
+//    rmwNode           - The node that has RMW semantics (if applicable)
+//    useRefPosition - The use refposition that need to be delay-freed.
+//
+void LinearScan::DelayFreeUses(RefPosition* useRefPosition, GenTree* rmwNode)
```

Comment:
Should this be called something like `AddDelayFreeUses` and called from `BuildDelayFreeUses` to ensure future refactorings/changes don't update one place but not the other?

Reply:
Sure, done.